docs/user/installation.rst

```diff
@@ -158,6 +158,8 @@ Navigate to your preffered installation directory, i. e. use ``mkdir ~/intelmq &
 
 You need to prepare some volumes & configs. Edit the left-side after -v, to change paths.
 
+Change ``redis_host`` to a running redis-instance. Docker will resolve it automatically.
+
```

Comment:
What does `Docker will resolve it automatically` mean?

Reply:
Docker do have an internal dns service, where you can expose services to each other without any extra configuration. I. e. "redis_host" will be resolved from docker dns as "172.0.1.1".